kotsadm/web/src/components/apps/DashboardCard.jsx

```diff
@@ -196,13 +196,15 @@ export default class DashboardCard extends React.Component {
   }
 
   renderLicenseCard = () => {
-    const { appLicense, isSnapshotAllowed } = this.props;
+    const { appLicense, isSnapshotAllowed, getingAppLicenseErrMsg } = this.props;
     const expiresAt = getLicenseExpiryDate(appLicense);
 
     return (
       <div>
         {isSnapshotAllowed ?
-          null
+          getingAppLicenseErrMsg ?
```

Comment:
this is difficult to read with all the `?` and `:`. for this line, how about something like this:
```
getingAppLicenseErrMsg && <p className="u-color--chestnut u-fontSize--small u-fontWeight--medium u-lineHeight--normal flex">{getingAppLicenseErrMsg}</p>
```